app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+  end
+
+  def show
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:id])
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    post_params = params.require(:post).permit(:title, :text)
```

Comment:
- [ ] [OPTIONAL] Good job on using strong parameters, but I think to make it more reusable then you can include it as a private method that can be called from anywhere inside the class.